core/domain/user_services.py

```diff
@@ -306,6 +310,12 @@ def update_user_bio(user_id, user_bio):
     _save_user_settings(user_settings)
 
 
+def set_contribution_date(user_id, first_contribution_datetime):
```

Comment:
actually, update_first_contribution_datetime. Try to follow existing style and conventions in the surrounding code.